### DISCUSSION TITLE
[Feature] shared albums view in mobile app

### DISCUSSION THREAD

**Discussion by wardwygaerts** [immich-app/immich#9003] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the web view, albums which are shared with my partner are displayed next to her other albums. Perfect!
In the mobile app, she only can see these albums in the Sharing tab in a list view. 
Is it possible to add a grid view in the sharing tab or to include these albums in the Library tab? Maybe than it's also possible the change the sorting because now it seems to be random.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by sidamos** [comment]
I'd also like to have that. It's way better like it is in the web view, having shared albums from another person along side all my own albums (shared or not).
